Analysis/Deserialize-KansaField.ps1

```diff
@@ -51,6 +51,11 @@ function GetTimestampUTC {
     Get-Date (Get-Date).ToUniversalTime() -Format "yyyy-MM-ddTHH:mm:ssZ"
 }
 
+if($((Get-WmiObject Win32_OperatingSystem).Name).Contains(" XP ")){
+Write-Error ("{0}: This script is not supported on Windows XP." -f (GetTimestampUTC))
```

Comment:
This may not work as expected for all localized versions of Windows. Also, XP and 2K3 are the same major.minor version number, but this check only looks for XP. Better to go by the major.minor version number returned by `(Get-CimInstance Win32_OperatingSystem).Version` and make sure that value is greater than 5.2. From memory, I can't recall if Get-CimInstance works on all windows OS versions, need to test.